media_player/alexa.py

```diff
@@ -496,8 +530,8 @@ def get_inputs(self, soup):
                 pass
         return data
 
-    def login(self, cookies=None, captcha=None):
-
+    def test_loggedin(self, cookies=None):
```

Comment:
This can eventually be converted to an auth decorator (probably when the AlexaAPI is moved to a pip module)

Reply:
Learned something new today about auth decorators. It probably needs to be for all the API functions as I suggested in #10. Agree that probably makes more sense in the pip module.